### PR TITLE
feat: update sql toolbar styling

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY
Styling of the toolbar according to designs

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
NEW: 
_DEV__Superset

### TEST PLAN
visual testing only (other test should still pass)

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/common/components/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import React from 'react';
 import { styled } from '@superset-ui/core';
 // eslint-disable-next-line no-restricted-imports
 import { Menu as AntdMenu, Dropdown, Skeleton } from 'antd';
-import { DropDownProps } from 'antd/lib/dropdown';
+import { DropDownProps as AnddDropdownProps } from 'antd/lib/dropdown';
```
Comment (hughhhh) [inline superset-frontend/src/common/components/index.tsx:23]:
```suggestion
import { DropDownProps as AntdDropdownProps } from 'antd/lib/dropdown';
```

File: superset-frontend/src/SqlLab/components/RunQueryActionButton.tsx [hunk 2]
```diff
@@ -17,75 +17,114 @@
...
+const buildText = (
+  shouldShowStopButton: boolean,
+  selectedText: string | undefined,
+): string | JSX.Element => {
+  if (shouldShowStopButton) {
+    return (
+      <>
+        <i
+          className="fa fa-stop"
+          style={{ display: 'inline-block', marginRight: '10px' }}
```
Comment (hughhhh) [inline superset-frontend/src/SqlLab/components/RunQueryActionButton.tsx:48]: can we set this as var above the build text vs inline

Comment (eschutho) [reply superset-frontend/src/SqlLab/components/RunQueryActionButton.tsx:48]: ah, thanks. I meant to come back to that.

**Comment by eschutho** [comment]
restarting cypress

**Review by hughhhh — APPROVED** [review]

**Pull request merged**